### PR TITLE
Add explanation of what `arguments` object is. 

### PR DESCRIPTION
**Pull request by aturkewi**
This pull request has been automatically created by learn.co.

### CODE REVIEW

**Comment by aturkewi** [comment]
@pletcher this is my pass at fixing issue #3

File: README.md [hunk 1]
```diff
@@ -72,6 +72,19 @@ Just omit the curly braces from around the function body.
 
 ## No More `arguments`
 
+So one thing you might not have known is that if you're inside a regular function, we get this seret object created for us called `arguments`. Let's see what that looks like:
```
Comment (pletcher) [inline README.md]: "seret" -> "secret"

File: README.md [hunk 2]
```diff
@@ -72,6 +72,19 @@ Just omit the curly braces from around the function body.
...
+
+```javascript
+function lotsOfArguments(arg1, arg2, arg3) {
```
Comment (pletcher) [inline README.md]: Would it be clearer without any arguments at all?

**Comment by pletcher** [comment]
Thanks for this, @aturkewi! Just a couple minor comments, and also: could we fence off this section with some sort of warning that students won't really be expected to know this for this course?

**Comment by aturkewi** [comment]
@pletcher Just removed entire section on arguments as it's no longer taught up to this point in the course.

**Pull request merged**